src/components/GettingStarted/GettingStarted.js

```diff
@@ -21,9 +23,31 @@ const Wrapper = styled.div`
 `;
 
 const GettingStarted = (props) => {
```

Comment:
Refactored this to compute all necessary info without needing props, so it could be used for both MAPI and REST implementations